### DISCUSSION TITLE
✅ 🔗 Repeater for FAQ/Toggle & Pop Up Dynamic with ACF capability (Duplicate of #5979)

### DISCUSSION THREAD

**Discussion by felmerald** [elementor/elementor#25377] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

I'm always frustrated when building a dynamic FAQ section using custom post type with ACF fields and Dynamic Pop-up most especially when having a team section using grid loop/carousel loop when clients want to show the pop-up for Biography. 

Currently Elementor Pro doesn't support this feature  and I see this has been requested multiple times

### Describe the solution you'd like.

It would be nice if the Elementor Pro version to start implementing the Repeater fields for FAQ/toggle with ACF fields capability and a Dynamic Pop-Up like Crocoblock Pop-up & jetEngine.

This is really important most especially for dynamic websites

### Additional context

Here's an example

Let's say we will create a custom post type "team-members" using ACF with the Custom Fields "name", "bio", "FAQ"
The FAQ is one of the problem here if the user doesn't know how to code a PHP,  because currently Elementor doesn't support a repeater functionality yet. 

Second, there is a carousel slider loop for team members and  the clients want a POP-UP for the BIO instead of routing them to its single post, currently the Elementor doesn't support for a dynamic pop-up with ACF fields.

Hopefully, Elementor can read this request

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there @felmerald!

thanks for submitting your feature request! 🙏 

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

### 🔗 🚀 For the ACF Repeater support, there is already a similar request submitted by someone here #5979

### 🔗 🚀 For the Dynamic Popup functionality, there is already a similar request submitted by someone here #21046

Although not exactly the same use case (Product Quick View), the goal is the same.

Feel free to continue the conversation there!

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.26.0! 🥳 

You can now use the **Off-Canvas** widget inside your **Loop Item** templates to display additional Dynamic Content for each post in your **Loop Grid** widgets! 🎉 🥳 🙌 

You can learn more about the Off-Canvas widget (and how to use it) here: 
https://elementor.com/help/off-canvas-widget/
And about the Loop Grid here: 
https://elementor.com/help/loop-grid/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
